lib/activator.js

```diff
@@ -177,11 +178,12 @@ module.exports = {
 		templates = config.templates || DEFAULTS.templates;
 		resetExpire = config.resetExpire || DEFAULTS.resetExpire;
 		proto = config.protocol || DEFAULTS.proto;
-		mailer = smtp(transport,templates, config.attachments);
+		mailer = smtp(transport,templates, config.styliner);
```

Comment:
I think the attachments was being passed there, but it wasn't actually used. Unit tests still run properly for all attachments cases.